src/plugins/intel_cpu/thirdparty/llmdnn/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.18)
+
+project(root)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+option(BUILD_TESTS "Build with tests" ON)
+
+message(INFO "--------------------------------")
+message(STATUS "Build with tests: ${BUILD_TESTS}")
+message(INFO "--------------------------------")
+
+set(CMAKE_CXX_STANDARD 17)
+if(MSVC)
+  # TODO
+  message(FATAL_ERROR "Not support yet. Use intel compiler 2023.0+.")
```

Comment:
let's just fix compilation with MSVC.
We cannot have a dependency on Intel compiler, because it gives extra overhead in binary size and new dependencies.

Reply:
Will do(added to TODO list).

Reply:
Remove the dependency of icc.